aws/resource_aws_athena_database.go

```diff
@@ -38,18 +38,73 @@ func resourceAwsAthenaDatabase() *schema.Resource {
 				Optional: true,
 				Default:  false,
 			},
+			"encryption_key": {
```

Comment:
The API refers to this as `encryption_configuration` -- is there any reason to diverge from their naming?

Reply:
I'd modeled it after the CodePipeline's HCL, which makes a bit more sense, but if the pattern is to copy the API directly to HCL I'll update.

Reply:
It might make a bit more sense today, but not necessarily in the future should the API change.  Here are a few reasons we prefer to stick with the API:
* We have gotten bitten in the past by trying to create abstractions and the API changes in a way that doesn't match those abstractions.
* Less of a maintenance burden to mentally work with the codebase if there are no abstractions/conversions.
* It keeps Terraform in line with the AWS API documentation, SDKs, and CloudFormation so operators coming from other documentation or systems can easily reference or convert between them.
* Different AWS services don't always line up with each other in terms of implementations. While this may look similar to CodePipeline today, Athena or CodePipeline may change tomorrow.

Hope this helps!